### PR TITLE
Mslq52 55

### PR DESCRIPTION
**Pull request by lowjunen**
off-the-top-of-my-head version. while complete, I feel that proper way to verify would be with an actual setup. 

This version also possesses updates to convention/terminology that we might want to standardize. hence the decision to throw it into review.

### CODE REVIEW

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [inline docs/source/software/quick_start_guide.rst]: Mission Control should be Ground Station or GS

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [inline docs/source/software/quick_start_guide.rst]: Should these also be links to the main website of the tools you're suggesting?

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [inline docs/source/software/quick_start_guide.rst]: adviced -> advised

File: docs/source/software/quick_start_guide.rst
Comment (exoticDFT) [inline docs/source/software/quick_start_guide.rst:43]: I would add a link to the section instead of saying "see next section". Since technically the next section is "Ground Station/Mission Control"

**Review by exoticDFT — CHANGES_REQUESTED** [review]
Couple of minor fixes, but otherwise it looks good.

**Review by k2shah — COMMENTED** [review]
is this page going to change? I feel like my recipes are being stolen from me without compensation.

File: docs/source/external_resources.rst
Comment (k2shah) [inline docs/source/external_resources.rst:5]: is this page going to change? I feel like my recipes are being stolen from me without compensation.

File: docs/source/software/quick_start_guide.rst
Comment (k2shah) [inline docs/source/software/quick_start_guide.rst:75]: are you sure everything is on 1.9?

**Review by k2shah — COMMENTED** [review]
is this `External Resources` going to change? I feel like my recipes are being stolen from me without compensation. 

also check the version, I haven't updated the firmware but if you have then lgtm

**Comment by harukins** [comment]
Looks good to me!

File: docs/source/references.rst
Comment (harukins) [inline docs/source/references.rst:8]: Links to PDF?

File: docs/source/software/quick_start_guide.rst
Comment (harukins) [reply docs/source/software/quick_start_guide.rst:75]: I think the current version of quads use v1.7.3 (or v1.7.4)

**Review by harukins — COMMENTED** [review]
A few comments added but otherwise lgtm.

File: docs/source/software/quick_start_guide.rst
Comment (lowjunen) [reply docs/source/software/quick_start_guide.rst:75]: hmmm... this might be a little problematic. the px4 wiki only goes back to v1.8. I have a quad with me that runs on v1.9 (wasn't on v1.10 as back then... their documentation was still work-in-progress). I believe switching to v1.9 would lose only the direct motor control from zijian's stuff?

File: docs/source/references.rst
Comment (lowjunen) [reply docs/source/references.rst:8]: can ignore as this was leftover before fork to alex's

File: docs/source/external_resources.rst
Comment (lowjunen) [reply docs/source/external_resources.rst:5]: leftover from prior fork. can ignore.

File: docs/source/software/quick_start_guide.rst
Comment (lowjunen) [reply docs/source/software/quick_start_guide.rst]: added the links

**Comment by exoticDFT** [comment]
Made a couple of changes to the formatting to fix a few minor things I saw. Otherwise I think this is good to go if there isn't any other feedback and the checks seem to pass.

**Pull request merged**
